sklearn/datasets/twenty_newsgroups.py

```diff
@@ -71,7 +71,7 @@
 TEST_FOLDER = "20news-bydate-test"
 
 
-def download_20newsgroups(target_dir, cache_path):
+def _download_20newsgroups(target_dir, cache_path):
```

Comment:
was this listed in classes.rst? Generally making anything private requires a deprecation cycle :-/

Reply:
No it's not listed anywhere in the doc, and it's not even exported in `datasets/__init__.py.`

Reply:
As far as I can tell it's only used as a helper for `fetch_20newsgroups`.